File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_entryrn.java
// EXPECTED: 55
// Reference emit for ENTRYRN — CALL quoted ENTRY alternate path.
public final class EntryrnRef {
  public static void main(String[] args) {
    String entry = "ALTPHASE";
    int phase = "ALTPHASE".equals(entry) ? 55 : 10;
    System.out.printf("%02d%n", phase);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_entryrn.cs
// EXPECTED: 55
// Reference emit for ENTRYRN — CALL quoted ENTRY alternate path.
using System;
public static class EntryrnRef {
  public static void Main() {
    string entry = "ALTPHASE";
    int phase = entry == "ALTPHASE" ? 55 : 10;
    Console.WriteLine(phase.ToString("D2"));
  }
}
